### DISCUSSION TITLE
Legacy comments: Baste-RainGames

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#447] [Announcements]
### Hi @Baste-RainGames! 🎉
We have put in the time to revive the [User Contributed Notes for Unity Docs](https://mvi.sh/notes/) extension, and it's now hosted using GitHub discussions managed by [giscus](https://github.com/giscus/giscus).  
You can read more about the new extension [here](https://github.com/Developer-Notes-Extension)!

### Legacy comments
These curated comments were ported from the original Disqus extension. They may be edited slightly to improve markdown and phrasing during the move! Many haven't survived due to being out of date, newly documented, or by being incompatible with the package docs.

**There are four comments that can be reposted under your account:**
- [Manual/SL-UnityShaderVariables](https://github.com/Developer-Notes-Extension/Unity-Notes/discussions/359#discussioncomment-3966068)
- [ScriptReference/Object.DontDestroyOnLoad](https://github.com/Developer-Notes-Extension/Unity-Notes/discussions/360#discussioncomment-3966069)
- [ScriptReference/AsyncOperation-allowSceneActivation](https://github.com/Developer-Notes-Extension/Unity-Notes/discussions/149#discussioncomment-3966070)
- [ScriptReference/MonoBehaviour.OnTriggerEnter](https://github.com/Developer-Notes-Extension/Unity-Notes/discussions/361#discussioncomment-3966071)

[✨ Click here](https://github.com/login/oauth/authorize?client_id=Iv1.34fef06c1f519b72&redirect_uri=https%3A%2F%2Fnotes.vertx.xyz%2Fport.html?state=aHR0cHMlM0ElMkYlMkZnaXRodWIuY29tJTJGRGV2ZWxvcGVyLU5vdGVzLUV4dGVuc2lvbiUyRlVuaXR5LU5vdGVzJTJGZGlzY3Vzc2lvbnMlMkZjYXRlZ29yaWVzJTJGbm90ZXM=) to repost the comments as yourself[^1].
👤 Reply to this discussion if you want them anonymized and we'll perform that action.
💤 Do nothing and leave the comments as they currently are.

[^1]: Comment porting is done via a stateless function using a GitHub app. No user tokens are stored.
If a comment already has replies it cannot be ported.

**Comment by Baste-RainGames** [comment]
You can repost them!

**Reply by vertxxyz** [reply]
Sorry, to clarify, would you like them anonymised or posted under your account?
You'll have to click the [porting link](https://github.com/login/oauth/authorize?client_id=Iv1.34fef06c1f519b72&redirect_uri=https%3A%2F%2Fnotes.vertx.xyz%2Fport.html?state=aHR0cHMlM0ElMkYlMkZnaXRodWIuY29tJTJGRGV2ZWxvcGVyLU5vdGVzLUV4dGVuc2lvbiUyRlVuaXR5LU5vdGVzJTJGZGlzY3Vzc2lvbnMlMkZjYXRlZ29yaWVzJTJGbm90ZXM=) if you'd like them as yourself.
(or if you find the app scary, manually repost them at the end of the respective threads and let me know you're done so so I can delete the bot's posts)

**Comment by Baste-RainGames** [comment]
I could've posted them under my own account but the porting link requires scary permissions ("act on my behalf"), so it's fine if you just post them anonymized.

**Reply by vertxxyz** [reply]
If you repost them manually (don't reply to the bot, post to the end of the threads) I am happy to clean up the bot's posts.

Sadly with the permissions, github seems to show that no matter the permissions enabled in the app, https://github.com/community/community/discussions/37117#discussioncomment-3973136 they intend it to mean that the app will be using your identity to perform whatever permissions it needs. In this case it's just:
- Read access to metadata (required)
- Read and write access to discussions
